io/net/net_data.py

```diff
@@ -161,10 +169,10 @@ def interface_wait(self, cmd):
         '''
          Waits for the interface to come up
         '''
-        for i in range(0, 600, 5):
+        for _ in range(0, 600, 5):
```

Comment:
Please notice that our `wait()` also accepts the `step` parameter, which I didn't use in my example. 
```
    wait_for(func, timeout, first=0.0, step=1.0, text=None)
        Wait until func() evaluates to True.
        
        If func() evaluates to True before timeout expires, return the
        value of func(). Otherwise return None.
        
        :param timeout: Timeout in seconds
        :param first: Time to sleep before first attempt
        :param step: Time to sleep between attempts in seconds
        :param text: Text to print while waiting, for debug purposes
```
The current test uses a 5s step, so you can reproduce the same behavior.